media_driver/agnostic/gen9_kbl/media_interfaces/media_interfaces_g9_kbl.h

```diff
@@ -197,7 +197,7 @@ class CodechalEncodeInterfacesG9Kbl
 #endif
 
 #ifdef _AVC_ENCODE_VME_SUPPORTED
-    using AvcEnc   = CodechalEncodeAvcEncG9Kbl;
+    using AvcEnc   = CodechalEncodeAvcEncG9Skl;
```

Comment:
Could we move this to CFL specific file?

Reply:
Sure. I have already modified the code to move the MFE enablement to CFL specific files. I will upload the patch later.